### PR TITLE
Ds xvi

### PR DESCRIPTION
**Pull request by karljonaskarlsson**

### CODE REVIEW

File: new/ds-xvi.xml [hunk 1]
```diff
@@ -0,0 +1,65 @@
+<?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0" xml:id="ds-xvi" xml:lang="en" type="mss">
+    <teiHeader>
+        <fileDesc>
+            <titleStmt>
+                <title>DS-XVI</title>
```
Comment (eu-genia) [inline new/ds-xvi.xml:8]: as title, we usually put the main content, not the shelfmark

File: new/ds-xvi.xml [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+        </profileDesc>
+        <revisionDesc>
+            <change who="JK" when="2022-04-08">Created entity</change>
+        </revisionDesc>
+    </teiHeader>
+    <text>
+        <body>
+            <div type="bibliography">
+                <listRelation>
+                    <relation name="dcterms:creator" active="ds-xvi" passive=""/>
```
Comment (eu-genia) [inline new/ds-xvi.xml:107]: please remove or complete the empty relation
I assume it was an error as dcterms:creator is a relation for authors of works, not for manuscripts

**Review by eu-genia — CHANGES_REQUESTED** [review]

File: new/ds-xvi.xml [hunk 3]
```diff
@@ -104,7 +104,6 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
         </ab></div>
             <div type="bibliography">
                 <listRelation>
```
Comment (eu-genia) [inline new/ds-xvi.xml:106]: you should also remove the now empty `listRelation `element

**Review by eu-genia — CHANGES_REQUESTED** [review]

**Review by eu-genia — APPROVED** [review]

**Comment by karljonaskarlsson** [comment]
I don't understand how those empty elements appeared. Did I add them by mistake somehow? Thanks for pointing out the errors!

**Comment by eu-genia** [comment]
In the create new ms form there is a list of possible relations, you must have clicked there by mistake, so the app generated the relation. Nevermind! 
(And we often do need relations, this way the app creates them in the correct place.)

**Comment by karljonaskarlsson** [comment]
Ok, it's good to know :)

**Pull request merged**